homeassistant/components/renson/services.yaml

```
@@ -0,0 +1,140 @@
+set_timer_level:
+  fields:
+    timer_level:
+      required: true
+      default: "level1"
+      selector:
+        select:
+          translation_key: "level_setting"
+          options:
+            - "level1"
+            - "level2"
+            - "level3"
+            - "level4"
+            - "holiday"
+            - "breeze"
+    time:
```

```suggestion
    minutes:
```

Imo you should use another name as `time` can be misleading as maybe a user expect to insert a time instead of a number. Also minutes makes it more clear